### PR TITLE
C++: Add a sample class in PrintAST.ql

### PR DESCRIPTION
**Pull request by jbj**
I've found myself typing out this class whenever I want to print the AST of one function. I hope it will be useful to others too.

Or is there a better way?

### CODE REVIEW

File: cpp/ql/src/semmle/code/cpp/PrintAST.ql [hunk 1]
```diff
@@ -7,3 +7,12 @@
 
 import cpp
 import PrintAST
+
+/**
+ * Temporarily tweak this class or make a copy to control which functions are
+ * printed.
+ */
+class Cfg extends PrintASTConfiguration {
+  /** Holds if the AST for `func` should be printed. */
```
Comment (zlaski-semmle) [inline cpp/ql/src/semmle/code/cpp/PrintAST.ql]: I would add `Tweak this predicate as needed`, perhaps in all caps.  Otherwise LGTM.

**Comment by jbj** [comment]
I'm happy to use all caps here, especially since I'm not the one who suggested it :). Done.

**Comment by dbartol** [comment]
What's the typical usage? Copy `PrintAST.ql` locally and tweak the condition?

**Comment by jbj** [comment]
I always tweak `PrintAST.ql` directly (and try to be careful not to commit my changes). For more long-lived uses, it would make sense to copy it.

**Pull request merged**